css/OWNERS

```diff
@@ -4,7 +4,12 @@
 {
   rules: [
     {
-      owners: [{ name: 'aghassemi' }, { name: 'camelburrito' }]
+      owners: [
+        { name: 'aghassemi' },
+        { name: 'camelburrito' },
```

Comment:
Delayed drive-by comment: We should probably add `requestReviews: false` to these two lines so that @aghassemi and @camelburrito are not auto-assigned as reviewers. I'll do this in #24971.

Reply:
good looking out!